MiniGames/MiniGames/round_result.cpp

```diff
@@ -0,0 +1,41 @@
+// round_result.cpp -- program for determining the winner
+
+
+int get_choice_from_symbol(char);
+
+
+static char round_result(char user_choice, int pc_choice)
+{
+	int const number_values = 3;
+
+	int user = -1;
+
+	user = get_choice_from_symbol(user_choice);
```

Comment:
get `choice` from symbol (`_choice`) that looks strange, need to think about namings a bit more)

Reply:
int user_number = convert_symbol(symbol);